lib/stdlib/test/erl_anno_SUITE.erl

```diff
@@ -463,6 +470,23 @@ primary_value(line, State) ->
                Line ->
                    Line
            end};
+primary_value(end_location, State) ->
+    case lists:keyfind(end_location, 1, State) of
+        false ->
+            case lists:keyfind(text, 1, State) of
+                false ->
+                    undefined;
+                {text, Text} ->
+                    case lists:keyfind(location, 1, State) of
+                        false ->
+                            undefined;
+                        {location, Location} ->
+                            {end_location, erl_anno:end_location(erl_anno:set_text(Text, erl_anno:new(Location)))}
```

Comment:
The `end_location` is a bit tricky in tests because it's both primary and secondary part of anno. In order to accommodate this, here we need to infer the value here (similarly to the clause for `line` above). Duplicating the whole text scan logic from the actual implementation sounds weird, so I call the inference directly.

I am happy to revise the tests, if another approach is preferable :)